packages/mds-providers/index.ts

```diff
@@ -177,17 +192,26 @@ export const providers: Readonly<{ [P in PROVIDER_ID]: Readonly<Provider> }> = O
     provider_id: TEST2_PROVIDER_ID,
     provider_name: 'Test 2'
   }),
-  [TEST3_PROVIDER_ID]: Object.freeze({
-    provider_id: TEST3_PROVIDER_ID,
-    provider_name: 'Test 3'
+  [MOCHA_PROVIDER_ID]: Object.freeze({
+    provider_id: MOCHA_PROVIDER_ID,
+    provider_name: 'Mocha Test Provider'
+  }),
+  [SUPPORT1_PROVIDER_ID]: Object.freeze({
+    provider_id: SUPPORT1_PROVIDER_ID,
+    provider_name: 'Support 1'
+  }),
+  [SUPPORT2_PROVIDER_ID]: Object.freeze({
+    provider_id: SUPPORT2_PROVIDER_ID,
+    provider_name: 'Support 2'
   }),
-  [TEST4_PROVIDER_ID]: Object.freeze({
-    provider_id: TEST4_PROVIDER_ID,
-    provider_name: 'Test 4'
+  [SUPPORT3_PROVIDER_ID]: Object.freeze({
+    provider_id: SUPPORT3_PROVIDER_ID,
+    provider_name: 'Support 3'
   }),
-  [TEST5_PROVIDER_ID]: Object.freeze({
-    provider_id: TEST5_PROVIDER_ID,
-    provider_name: 'Test 5'
+  [DEPRECATED_BLUE_SYSTEMS_PROVIDER_ID]: Object.freeze({
+    provider_id: BLUE_SYSTEMS_PROVIDER_ID,
+    provider_name: 'Blue Systems',
```

Comment:
Might be worth making the `provider_name` `Blue Systems (DEPRECATED)` to make look-back easier